src/PhpSessionPersistence.php

```diff
@@ -105,4 +147,82 @@ private function generateSessionId() : string
     {
         return bin2hex(random_bytes(16));
     }
+
+    /**
+     * Generate cache headers for a given session cache_limiter value.
+     * @param string $cacheLimiter
+     * @param int $cacheExpire
```

Comment:
Parameter annotations are discouraged unless they add descriptions; these can be removed.

Reply:
do I keep the method's purpose explaination?

Reply:
Yes.